stage-tipi/01-config/files/app/setup.py

```
@@ -425,11 +425,17 @@ def get_final_ip() -> str | None:
 def configure_cockpit(enabled: bool):
     if enabled:
         step(T["cockpit_step"])
+        # Le socket et le service sont masqués au build — il faut d'abord
+        # lever le masque avant de pouvoir les activer.
+        subprocess.run(["systemctl", "unmask", "cockpit.socket", "cockpit.service"],
```

Ruff (0.15.15)

[error] Starting a process with a partial executable path

(S607)